ultra/train.py

```diff
@@ -146,13 +152,28 @@ def train(task, num_episodes, policy_class, eval_info, timestep_sec, headless, s
     parser.add_argument(
         "--seed", help="environment seed", default=2, type=int,
     )
+    parser.add_argument(
+        "--log-dir", help="log directory location", default="logs", type=str,
+    )
     args = parser.parse_args()
 
     num_cpus = max(
         1, psutil.cpu_count(logical=False) - 1
     )  # remove `logical=False` to use all cpus
 
-    policy_class = "ultra.baselines.sac:sac-v0"
+    with open("/home/jenish/Desktop/temp/SMARTS/ultra/agent_pool.json", "r") as f:
```

Comment:
Oh maybe this hardcoded path to the agent_pool.json file has to be changed?

Reply:
Smh, I actually forgot about that lol